src/app/profile/settings/feature-opt-in/feature-opt-in.component.html

```diff
@@ -0,0 +1,95 @@
+<div class="container-fluid container-cards-pf">
+  <div class="col-xs-10 col-sm-6 col-md-6">
+    <div class="card-pf list-group form-group list-view-pf list-view-pf-view">
+      <div class="row">
+        <div class="list-group-item">
+          <div class="list-view-pf-checkbox">
+            <input type="radio" name="No Pre-Production Features" value="released" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+          </div>
+          <div class="list-view-pf-main-info">
+            <div class="list-view-pf-left">
+              <span class="fa fa-globe list-view-pf-icon-sm"></span>
+            </div>
+            <div class="list-view-pf-body">
+              <div class="list-view-pf-description">
+                <div class="list-group-item-heading">
+                  Production Only Features
+                </div>
+                <div class="list-group-item-text">
+                  This is the default and current release version of the product.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="list-group-item">
+          <div class="list-view-pf-checkbox">
+            <input type="radio" name="Beta Features" value="beta" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+          </div>
+          <div class="list-view-pf-main-info">
+            <div class="list-view-pf-left">
+              <span class="fa list-view-pf-icon-sm" style="font-family: 'Open Sans'; font-size:18px; line-height: 1.3"> ß </span>
```

Comment:
Please do not use inline styles - create a class and apply these styles to that class.